src/main/resources/liquibase/harness/data/expectedResultSet/hsqldb/loadUpdateData.json

```diff
@@ -0,0 +1,60 @@
+{
+  "loadUpdateData": [
```

Comment:
i'm afraid this file won't work as time will change for each run. For this test you can skip expected json